Makefile

```diff
@@ -35,20 +37,23 @@ utils = github.com/golang/protobuf/protoc-gen-go \
 		github.com/spiffe/spire/tools/protoc-gen-spireplugin
 
 # Help message settings
-cyan := $(shell tput setaf 6)
-reset := $(shell tput sgr0)
-bold  := $(shell tput bold)
+cyan := $(shell which tput && tput setaf 6 || echo "")
```

Comment:
Great! Maybe another improvement could be to redirect `which` output to `/dev/null`:
```
cyan := $(shell which tput > /dev/null && tput setaf 6 || echo "")
```
Otherwise `which` output will be printed when `make help` is run and `tput` exists

(Same for `reset` and `bold`)

Reply:
Good catch! Thanks.